src/Flow/Mongo/MongoFile.php

```diff
@@ -90,12 +90,22 @@ public function saveChunk()
             ) {
                 throw new \Exception("Invalid upload! (size: {$actualChunkSize})");
             }
-            $chunk['data'] = new \MongoBinData($data);
+            $chunk['data'] = new \MongoBinData($data, 0); // \MongoBinData::GENERIC is not defined for older mongo drivers
             $this->config->getGridFs()->chunks->findAndModify($chunkQuery, $chunk, [], ['upsert' => true]);
             unlink($file['tmp_name']);
 
+            $this->ensureIndices();
+
             return true;
         } catch (\Exception $e) {
+            error_log("Could not store chunk: " . $e->getMessage() . "\n" . $e->getTraceAsString());
```

Comment:
To me it is meaningful, as something unexpected happened (db down, ...). If this violates coding rules for this project, I can of course remove it.